rllib/evaluation/tests/test_trajectory_view_api.py

```diff
@@ -129,7 +129,7 @@ def test_traj_view_lstm_prev_actions_and_rewards(self):
             view_req_policy = policy.view_requirements
             # 7=obs, prev-a + r, 2x state-in, 2x state-out.
             assert len(view_req_model) == 7, view_req_model
-            assert len(view_req_policy) == 22, (len(view_req_policy), view_req_policy)
+            assert len(view_req_policy) == 23, (len(view_req_policy), view_req_policy)
```

Comment:
Did this change by 1 because we added a new sample_batch key?

Reply:
Yup, correct.